tests/dash/dash_utils.py

```diff
@@ -0,0 +1,115 @@
+import json
+import logging
+
+from jinja2 import Template
+from os import path
+from time import sleep
+from tests.common.helpers.assertions import pytest_assert
+from constants import TEMPLATE_DIR
+
+logger = logging.getLogger(__name__)
+
+
+def safe_open_template(template_path):
+    """
+    Safely loads Jinja2 template from given path
+
+    Note:
+        All Jinja2 templates should be accessed with this method to ensure proper garbage disposal
+
+    Args:
+        template_path: String containing the location of the template file to be opened
+
+    Returns:
+        A Jinja2 Template object read from the provided file
+    """
+
+    with open(template_path) as template_file:
```

Comment:
Here it will fail in my case, because path is not absolute.
You can to build abs path to template dir:
```
full_path = path.join(path.dirname(path.abspath(__file__).split('dash')[0]), template_path)
with open(full_path...
```

Reply:
I have changed the 'TEMPLATE_DIR' variable in `constants.py` to be an absolute path, please check it